apps/scouting/backend/src/game-object-coral.ts

```diff
@@ -0,0 +1,17 @@
+import { GameObject } from "./game-object";
+import { AlgaeEvent, AllPossibleGameEvents, CoralEvent } from "./game-events";
+
+export function calculatePointsCoral(coral: GameObject<CoralEvent>): number {
+        return coral.gameEvents.L1*L1_SCORING+
+        coral.gameEvents.L2*L2_SCORING+
+        coral.gameEvents.L3*L3_SCORING+
+        coral.gameEvents.L4*L4_SCORING
+    }
+export function calculateRPCoral(): number {
```

Comment:
shouldn't this get a coral object?
```suggestion
export function calculateRPCoral(coral: GameObject<CoralEvent>): number {
```